R11BRT-1965 - remove support versions (#155)
OutSystems/OutSystems.SetupTools: examples/DeployOnPrem/README.md

4. Run the script ( .\\<scriptName.ps1> [parameters] )
    * The scripts takes two parameters. *-MajorVersion* and *-InstallDir*.

>     * *MajorVersion* is the major OutSystems version you want to install. This can be "11" and its a mandatory parameter.

Reviewer: ```suggestion
    * *MajorVersion* is the major OutSystems version you want to install. This must be "11" and it is a mandatory parameter.
```

4. Run the script ( .\\CreateOfflineBundle.ps1> [parameters] ).
    * The scripts takes one parameters. *-MajorVersion*.

>     * *MajorVersion* is the major OutSystems version that will be downloaded to the bundle. This can be "11"and its a mandatory parameter.

Reviewer: ```suggestion
    * *MajorVersion* is the major OutSystems version that will be downloaded to the bundle. This must be "11" and it is a mandatory parameter.
```
